### PR TITLE
Correcting and improving Portuguese locale

### PR DESCRIPTION
**Pull request by maiconkkl**

### CODE REVIEW

File: locales/br.lua [hunk 1]
```diff
@@ -33,14 +33,14 @@ Locales['br'] = {
...
-  ['take_stock'] = 'take Stock',
-  ['boss_actions'] = 'boss Actions',
-  ['mission_complete'] = 'mission Completed',
-  ['quantity'] = 'quantity',
-  ['quantity_invalid'] = 'that is an invalid quantity!',
-  ['inventory'] = 'inventory',
-  ['taxi_client'] = 'taxi Client',
+  ['deposit_stock'] = 'depositar estoque',
+  ['take_stock'] = 'retirar estoque',
+  ['boss_actions'] = 'Ações de chefe',
```
Comment (ElPumpo) [inline locales/br.lua]: upper case whyyyyyyy

File: locales/br.lua [hunk 2]
```diff
@@ -33,14 +33,14 @@ Locales['br'] = {
   ['must_in_vehicle'] = 'Você deve estar em um veículo para começar a missão',
   ['have_earned'] = 'Você ganhou ~g~$%s~s~',
```
Comment (ElPumpo) [inline locales/br.lua]: fix these too plz

File: locales/br.lua (hunk 1 above)
Comment (maiconkkl) [reply locales/br.lua]: Fixed

File: locales/br.lua (hunk 2 above)
Comment (maiconkkl) [reply locales/br.lua]: Fixed

**Pull request merged**